src/index.js

```diff
@@ -58,6 +59,16 @@ var handlers = {
 
     'GetReviewIntent': getReview,
 
+    'GetLatestReviewsIntent': getLatestReviews,
+
+    'ReviewTypeIntent': function() {
+        const slots = this.event.request.intent.slots;
+        if (slots.review_type && slots.review_type.value) {
+            this.event.session.attributes.reviewType = slots.review_type.value;
+            this.emit(this.event.session.attributes.lastIntent)
```

Comment:
I think we can make this more explicit and call `getLatestReviews` instead of `lastIntent`